addOns/kotlin/kotlin.gradle.kts

```
@@ -0,0 +1,27 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+
+plugins {
+    kotlin("jvm") version "1.3.50"
+}
+
+version = "1.0.0"
+description = "Allows Kotlin to be used for ZAP scripting - (some) templates included"
+
+zapAddOn {
+    addOnName.set("Kotlin Support")
+    addOnStatus.set(AddOnStatus.ALPHA)
+    zapVersion.set("2.9.0")
+
+    manifest {
+        author.set("StackHawk Engineering")
+    }
+}
+
+dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    implementation("org.jetbrains.kotlin:kotlin-compiler-embeddable")
+    implementation("org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable")
+    implementation("org.jetbrains.kotlin:kotlin-script-util")
```

Based on https://github.com/JetBrains/kotlin/pull/2057#issuecomment-628538983 we should probably avoid using this one, and rely directly on JSR223 dependencies.

```suggestion
    val ktVersion = "1.3.72"
    api("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$ktVersion")
    implementation("org.jetbrains.kotlin:kotlin-scripting-jsr223-embeddable:$ktVersion")
    implementation("org.jetbrains.kotlin:kotlin-compiler-embeddable:$ktVersion")
```